ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -535,6 +680,23 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
         </div>
       );
     },
+    closeTaxonomyPanel: function() {
+      this.props.closeTaxonomyPanel();
+    },
+    componentWillReceiveProps: function(nextProps) {
```

Comment:
What is the purpose of putting `removePanelVisibility` in the state? Why not just use `this.props.removePanelVisibility` in the `render` method?